src/webxr/ARButton.ts

```diff
@@ -1,28 +1,37 @@
+import { WebGLRenderer, XRSession, XRSessionInit } from 'three'
+
+interface ARButtonSessionInit extends XRSessionInit {
```

Comment:
Lets export this type, even if no one uses it, we might aswell.

Reply:
will do!